lib/collection/src/config.rs

```diff
@@ -56,6 +56,12 @@ pub struct CollectionParams {
     /// Number of replicas for each shard
     #[serde(default = "default_replication_factor")]
     pub replication_factor: NonZeroU32,
+    /// Defines how many replicas should apply the operation for us to consider it successful.
+    /// Increasing this number will make the collection more resilient to inconsistencies, but will
+    /// also make it fail if not enough replicas are available.
+    /// Does not have any performance impact.
+    #[serde(default = "default_concern_factor")]
+    pub concern_factor: NonZeroU32,
```

Comment:
IMHO `concern_factor` is not a descriptive name for this setting, users will have a hard time guess what it does.

I propose instead the following alternatives:
- `write_concern_factor`
- `write_consistency_factor`

Reply:
ok